force-app/main/default/objects/Template_task__c/fields/Task__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Task__c</fullName>
```

Comment:
Having a field called `Task` on an object called `Template Task` may be cause for some confusion. And since the same field is called `Requirements` on your `Journey Task` object, it would make sense for them to share the same field name.

Reply:
You said you had already addressed this but it seems like that isn't reflecting in the GitHub repo.